test/sequential/test-crypto-timing-safe-equal.js

```diff
@@ -23,12 +23,15 @@ assert.strictEqual(
 
 assert.throws(function() {
   crypto.timingSafeEqual(Buffer.from([1, 2, 3]), Buffer.from([1, 2]));
-}, 'should throw when given buffers with different lengths');
+}, /^TypeError: Input buffers must have the same length$/,
+   'should throw when given buffers with different lengths');
```

Comment:
Seems right to me. I think it's just a visual effect because of the `/` and the `'`.